bang/models.py

```diff
@@ -1511,133 +1517,120 @@ def __unicode__(self):
 
-    depends_on_life = models.PositiveIntegerField(null=True)
-    life = property(lambda _s: _s.depends_on_life)
+    values=models.CharField(max_length=100, null=True, help_text='Seperate with spaces in ascending order')
+    lifes=models.CharField(max_length=100, null=True, help_text='Seperate with spaces in ascending order')
 
-    ###
+    about = models.TextField(_('About'), null=True)
+    ABOUTS_CHOICES = ALL_ALT_LANGUAGES
+    d_abouts = models.TextField(_('About'), null=True)
 
     @property
     def formatted_name(self):
-        template = self.type_name_template
-        if not template:
-            return unicode(self.t_name or self.t_type or u'{}: #{}'.format(_('Area items'), self.id))
-        return template.format(**{
-            variable: unicode(getattr(self, variable, ''))
-            for variable in (templateVariables(template) or [])
-        })
-
-    @property
-    def affected_members(self):
-        if self.i_band is not None:
-            return self.t_band
-        elif self.i_attribute is not None:
-            return self.t_attribute
-        return _('All')
-
-    @property
-    def effect_value(self):
-        if self.flat:
-            return unicode(int(self.value))
-        return '{percentage}%'.format(percentage=self.value)
-
-    @property
-    def affected_stat(self):
-        if self.i_stat is not None:
-            return unicode(self.t_stat).lower()
-        return u'/'.join([unicode(stat).lower() for n, stat in self.STAT_CHOICES])
-
-    @property
-    def formatted_description(self):
-        template = (
-            _('{affected_members} members get +{effect_value} {affected_stat} points if life is above {life}')
-            if self.depends_on_life
-            else _('{affected_members} members get +{effect_value} {affected_stat} points')
-        )
-        return template.format(**{
-            variable: unicode(getattr(self, variable, ''))
-            for variable in (templateVariables(template) or [])
-        })
+        formatted_name=''
+        if unicode(self.member) != u'None':
+            if self.type in ['studio', 'poster', 'entrance']:
+                if self.type == 'studio' and self.instrument not in [None, 'mic']:
+                    formatted_name+=unicode(_('{name}\'s').format(name=unicode(self.member.first_name)))
+                else:
+                    formatted_name+=unicode(self.member.t_band)
+        if self.t_name:
+            formatted_name +=' ' + unicode(self.t_name)
+        if self.instrument not in ['other', None]:
+            formatted_name+=' ' + unicode(self.t_instrument)
+        return formatted_name or ''
+
+    @property
+    def affected(self):
+        string = ''
+        if self.i_attribute != None:
+            string += unicode(self.t_attribute)
+        if self.member != None:
+            if string != '':
+                string +=' '
+            string += self.member.t_band
+        return string if string != '' else None
```

Comment:
This method can be simplified down to 
```python
return u'{attr} {band}'.format(attr=self.t_attribute or '', 
    band=self.member.t_band if self.member else '').strip()
```

so you don't have to fiddle with spacing.